packages/rest/src/-private/builders/save-record.ts

```diff
@@ -36,10 +36,10 @@ function isExisting(identifier: StableRecordIdentifier): identifier is StableExi
  * const person = this.store.peekRecord('person', '1');
  *
  * // mark record as deleted
- * store.deleteRecord(person);
+ * this.store.deleteRecord(person);
```

Comment:
we should make all examples use just `store` and only use `this.store` when in the example we are inside a class method